src/profile/ChangePassword.js

```diff
@@ -11,91 +11,91 @@ import LoaderButton from "../components/LoaderButton";
  * @TODO refactor into React Hooks
  */
 
-export default class ChangePassword extends Component {
-  constructor(props) {
-    super(props);
+const ChangePassword = (props) => {
 
-    this.state = {
-      password: "",
-      oldPassword: "",
-      isChanging: false,
-      confirmPassword: "",
-    };
-  }
+  const [state, setstate] = useState({
```

Comment:
Would you please change `setstate` to `setState`? It is 'proper' to use camelCase when describing function names.

Reply:
I have made the variable naming to camel case.